### PR TITLE
Fix styling issues in images

### PR DESCRIPTION
**Pull request by anupama-pathirage**
## Purpose

Fixes https://github.com/wso2-enterprise/choreo/issues/3806 
Fixes https://github.com/wso2-enterprise/choreo/issues/4590
This will improve the styling of the images.
- Will add a boarder for all the images
- Resize depending on their usage- to 90% width for full images and 50% width for half images
- Will add a alt text for all the images

**Before the Fix:**

image

image

**After the Fix:**
image

image

### CODE REVIEW

**Review by shaniR — CHANGES_REQUESTED** [review]

**Comment by shaniR** [comment]
I could not view the images on github, but went by the screenshot that's added.

File: en/docs/apis/create-and-publish-your-first-api.md [hunk 1]
```diff
@@ -67,14 +67,14 @@ Follow the procedure below to publish the API and view the published API on the
...
 
 3. Click **Go to Devportal**.
 
-       ![](../assets/img/apis/go-to-devportal.png)
+       ![Go to DevPortal](../assets/img/apis/go-to-devportal.png){.cInlineImage-full}
 
      You can see the published API in the Choreo Devportal.
 
-       ![](../assets/img/apis/devportal.png)
+       ![DevPortal](../assets/img/apis/devportal.png){.cInlineImage-full}
```
[the remark this answers is not in the document]
Comment (anupama-pathirage) [reply en/docs/apis/create-and-publish-your-first-api.md]: Changed as `List of published APIs in the API Developer Portal`

**Comment by anupama-pathirage** [comment]
This PR has added only the missing alt texts. Created https://github.com/wso2-enterprise/choreo/issues/4726 to track the other alt text improvements.

**Comment by anupama-pathirage** [comment]
> I could not view the images on github, but went by the screenshot that's added.

Couldn't figure out an exact reason for not rendering images in github view option. Some files (e.g. create-your-first-service.md can be viewed)

Seems the issue is not related to the changes in this PR. Opened and verified via a dummy PR https://github.com/wso2/docs-choreo-dev/pull/103/files

**Review by RukshiW — APPROVED** [review]

**Comment by shaniR** [comment]
> > I could not view the images on github, but went by the screenshot that's added.
> 
> Couldn't figure out an exact reason for not rendering images in github view option. Some files (e.g. create-your-first-service.md can be viewed)
> 
> Seems the issue is not related to the changes in this PR. Opened and verified via a dummy PR https://github.com/wso2/docs-choreo-dev/pull/103/files

Since you mentioned, I just checked the difference, and it seems that it is caused by unwanted indentation. Shall we remove them?

**Comment by anupama-pathirage** [comment]
> > > I could not view the images on github, but went by the screenshot that's added.
> > 
> > 
> > Couldn't figure out an exact reason for not rendering images in github view option. Some files (e.g. create-your-first-service.md can be viewed)
> > Seems the issue is not related to the changes in this PR. Opened and verified via a dummy PR https://github.com/wso2/docs-choreo-dev/pull/103/files
> 
> Since you mentioned, I just checked the difference, and it seems that it is caused by unwanted indentation. Shall we remove them?

@shaniR Thanks for the suggestion. Yes it was the root cause. Fixed them all too. Please review.

File: en/docs/services/create-your-first-service.md [hunk 2]
```diff
@@ -16,15 +16,15 @@ To create the service, follow the procedure below:
...
+    ![Resource Configuration](../assets/img/services/configure-api-trigger.png){.cInlineImage-half}
     
     Then click **Save API**. Now you have saved the API trigger that starts your service.
     
 6. Click **API Calls**, and then enter information as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
 
     1. Click **HTTP**.
     
-        ![Select HTTP Connection](../assets/img/services/select-http-connection.png)
+        ![Select HTTP Connection](../assets/img/services/select-http-connection.png){.cInlineImage-half}
```
Comment (shaniR) [inline en/docs/services/create-your-first-service.md:27]: This image appears quite large. can we make it as small as possible, without causing any visual inconvenience?

File: en/docs/services/create-your-first-service.md [hunk 3]
```diff
@@ -37,15 +37,15 @@ To create the service, follow the procedure below:
         
     3. Click the **+** icon below the HTTP API call you added. 
     
-        ![Update Existing Connection](../assets/img/services/update-existing-connection.png)
+        ![Update Existing Connection](../assets/img/services/update-existing-connection.png){.cInlineImage-half}
         
     4. Click **API Calls**, and then click on the existing connection.
     
-        ![Select Existing connection](../assets/img/services/select-existing-connection.png)
+        ![Select Existing connection](../assets/img/services/select-existing-connection.png){.cInlineImage-half}
```
Comment (shaniR) [inline en/docs/services/create-your-first-service.md:44]: same here. can we resize this image to the smallest possible size, without causing any visual inconvenience?

File: en/docs/services/create-your-first-service.md [hunk 4]
```diff
@@ -37,15 +37,15 @@ To create the service, follow the procedure below:
...
         
     5. Enter information as follows:
         
-        ![Configure HTTP Connection](../assets/img/services/select-existing-connection.png)
+        ![Configure HTTP Connection](../assets/img/services/select-existing-connection.png){.cInlineImage-half}
```
Comment (shaniR) [inline en/docs/services/create-your-first-service.md:48]: same here

File: en/docs/services/create-your-first-service.md [hunk 5]
```diff
@@ -100,23 +101,23 @@ To create the service, follow the procedure below:
     
 9. To add a statement of the `Respond` type so that the result of the above processing is returned as a response, click the last **+** icon in your low-code diagram.
 
-    ![Add Respond Statement](../assets/img/services/add-respond-statement.png)
+    ![Add Respond Statement](../assets/img/services/add-respond-statement.png){.cInlineImage-half}
```
Comment (shaniR) [inline en/docs/services/create-your-first-service.md:104]: This image is too lengthy too. can we follow the same as suggested above?

File: en/docs/services/create-your-first-service.md [hunk 6]
```diff
@@ -100,23 +101,23 @@ To create the service, follow the procedure below:
...
     
     In the **Respond Expression** field, enter `response`, and then click **Save**.
 
     
 Now you have completed designing your service. It looks as follows.
 
-![No Code View](../assets/img/services/choreo-service-low-code-view.png)
+![No Code View](../assets/img/services/choreo-service-low-code-view.png){.cInlineImage-half}
```
Comment (shaniR) [inline en/docs/services/create-your-first-service.md:111]: same here

**Review by shaniR — APPROVED** [review]

File: en/docs/services/create-your-first-service.md (hunk 2 above)
Comment (anupama-pathirage) [reply en/docs/services/create-your-first-service.md:27]: Image size is not visible in github view mode. This shows the original image size.

File: en/docs/services/create-your-first-service.md (hunk 4 above)
Comment (anupama-pathirage) [reply en/docs/services/create-your-first-service.md:48]: Image size is not visible in github view mode. This shows the original image size.

File: en/docs/services/create-your-first-service.md (hunk 3 above)
Comment (anupama-pathirage) [reply en/docs/services/create-your-first-service.md:44]: Image size is not visible in github view mode. This shows the original image size.

File: en/docs/services/create-your-first-service.md (hunk 5 above)
Comment (anupama-pathirage) [reply en/docs/services/create-your-first-service.md:104]: Image size is not visible in github view mode. This shows the original image size.

File: en/docs/services/create-your-first-service.md (hunk 6 above)
Comment (anupama-pathirage) [reply en/docs/services/create-your-first-service.md:111]: Image size is not visible in github view mode. This shows the original image size.

**Pull request merged**